imports/plugins/core/ui/client/components/divider/verticalDivider.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export default function VerticalDivider() {
+  return (
+    <div
+      style={{
+        height: "20px",
+        width: 1,
+        backgroundColor: "#E6E6E6",
```

Comment:
Same comment as above, throw this into a LESS file? Or is this step 1 of CSS in JS?